src/http/server/handlers/create_query.rs

```diff
@@ -1,22 +1,19 @@
-use crate::helpers::{
-    query::QueryCommand,
-    transport::{
-        http::{
-            server::{handlers::QueryConfigFromReq, Error},
-            CreateQueryResp,
-        },
-        TransportCommand,
+use crate::{
+    helpers::{query::QueryCommand, transport::TransportCommand, CommandEnvelope, CommandOrigin},
+    http::{
+        server::{handlers::QueryConfigFromReq, Error},
+        CreateQueryResp,
     },
-    CommandEnvelope, CommandOrigin, HelperIdentity,
 };
 use axum::{routing::post, Extension, Json, Router};
 use hyper::{Body, Request};
 use tokio::sync::{mpsc, oneshot};
 
-#[cfg_attr(feature = "enable-serde", derive(serde::Deserialize))]
-struct CreateQueryBody {
-    helper_positions: [HelperIdentity; 3],
-}
+// TODO: do we need this?
```

Comment:
if it is not used, probably not?

Reply:
ok. it will be easy enough to add back if we ever do need it